src/pages/billing/Project.js

```diff
@@ -384,7 +384,8 @@ const ProjectDetail = ({ billingProject, reloadBillingProject, billingAccounts,
         h(Link, {
           tooltip: 'Remove Billing Account',
           style: { marginLeft: '0.5rem' },
-          disabled: billingProject.billingAccount === 'null',
+          // For some reason the api sometimes returns string null, and sometimes returns no field, and soemtimes returns null. This is just to be complete.
```

Comment:
should we create a tech debt ticket to fix this in the backend and add that ticket # here?

Reply:
https://broadworkbench.atlassian.net/browse/CA-1586 done

Reply:
thanks for creating the ticket! What do you think about adding it to the comment in the code, for ease of reference? I'm worried someone might read this line and try to create a (duplicate) ticket or need to go fishing for it in the great ocean that is jira.